init

```diff
@@ -30,9 +31,23 @@ function setup_virtualenv () {
     popd > /dev/null
 }
 
-if [[ $# -gt 1 ]]; then
+wsl_quicklisp_proxy=""
+if [[ $# -gt 2 ]]; then
     print_help
     exit 1
+elif  [[ $# -eq 2 ]]; then
+    case "$1" in
+      --wsl-quicklisp-proxy)
+          shift
+          wsl_quicklisp_proxy=":proxy \"http://$1/\""
+          shift
+      ;;
+      *)
+          # unknown option
+          print_help
+          exit 1
+      ;;
+    esac
 elif  [[ $# -eq 1 ]]; then
```

Comment:
First of all, I think this can be merged at this stage. However I think we should write our new scripts in python, as the easiest things are pretty nasty in bash (e.g.: handling parameters) compared to Python.